File: Java/Q3.java
import java.util.Scanner;

class Main {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        System.out.print("Enter the password: ");
        String password = scanner.nextLine();
        if (password.equals("admin123")) {
            System.out.println("Welcome");
        }
        scanner.close();
    }
}

File: CPP/Q3.cpp
#include <iostream>
#include <string>

int main() {
    std::string password;
    std::cout << "Enter the password: ";
    std::cin >> password;
    if (password == "admin123") {
        std::cout << "Welcome" << std::endl;
    }
    return 0;
}